vite.config.ts

```diff
@@ -6,38 +6,77 @@ import dts from "vite-plugin-dts";
 import svgr from "vite-plugin-svgr";
 
 export default defineConfig({
-  publicDir: "res",
   build: {
     target: browserslistToEsbuild(),
     assetsInlineLimit: 0, // To disable assets inlining as base64 in the bundle
-    lib: {
-      entry: resolve(__dirname, "src/index.ts"),
-      name: "@vector-im/compound-web",
-      fileName: "compound-web",
-    },
+
+    // We expect to be used as a dependency with a bundler, so we don't need to
+    // minify and let the bundler deal with the sourcemaps
+    sourcemap: true,
+    minify: false,
+    cssMinify: false,
+
     rollupOptions: {
-      external: ["react", "react-dom"],
+      // We are *not* running in library mode because inlines all the assets.
+      // Instead, we're running in normal mode & specifying the entrypoint here.
```

Comment:
Let's add a link to the vite issue that tracks this, https://github.com/vitejs/vite/issues/3295

Reply:
Done